apps/management-app/src/components/Common/FormComponents.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  TextField,
+} from '@mui/material';
+
+export function ReadOnlyTextField({
+  label,
+  value,
+  disabled = false,
+}: {
+  label?: string;
+  value?: string | number | undefined;
+  disabled?: boolean;
+}) {
+  return (
+    <TextField
+      label={label}
+      fullWidth
+      InputProps={{
+        readOnly: true,
+      }}
+      value={value}
+      disabled={disabled}
+    />
+  );
+}
+
+export function ReadOnlyTextForm({
+  label,
```

Comment:
```suggestion
  label = '',
```
should add default value when it's optional field